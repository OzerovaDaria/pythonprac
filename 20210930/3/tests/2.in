4,-1,3
4,-2,-6
2,0,3
5,3,-7
-1,6,-3
2,-4,1
