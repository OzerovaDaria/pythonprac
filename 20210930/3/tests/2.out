[27, -6, -22]
[10, 24, -28]
[16, -6, -11]
